addons/website_event/views/event_templates_list.xml

```diff
@@ -464,4 +464,10 @@
     </xpath>
 </template>
 
+<!-- Empty Placeholder used by website_event_track to show message on
+    website_event_sale confirmation page without having to add a bridge module -->
+<template id="event_confirmation_track_redirection_message">
```

Comment:
Don't like the "track" naming, it is not really modular ^^ could be something more generic, like ``event_confirmation_end_page_hook`` ? that way we know it is a hook to add stuff in the confirmation page, whatever we put inside.